
# HG changeset patch
# User Keir Fraser <keir.fraser@citrix.com>
# Date 1283332925 -3600
# Node ID 16867267ac126043712703455b81c16f2549ee23
# Parent e854f11d392d13e5302a219bcc9e6e352a4f1c1b
x86 intel: Disable XSAVE support.

It breaks HVM save/restore.

Signed-off-by: Keir Fraser <keir.fraser@citrix.com>
xen-unstable changeset:   ae0cd4e5cc01
xen-unstable date:        Wed Sep 01 10:19:14 2010 +0100

--- a/xen/arch/x86/cpu/intel.c	Wed Sep 01 10:21:41 2010 +0100
+++ b/xen/arch/x86/cpu/intel.c	Wed Sep 01 10:22:05 2010 +0100
@@ -30,7 +30,7 @@ integer_param("cpuid_mask_ecx", opt_cpui
 integer_param("cpuid_mask_ecx", opt_cpuid_mask_ecx);
 integer_param("cpuid_mask_edx", opt_cpuid_mask_edx);
 
-static int use_xsave = 1;
+static int use_xsave;
 boolean_param("xsave", use_xsave);
 
 #ifdef CONFIG_X86_INTEL_USERCOPY

